File: 251002/Troitsky/Lab1-5/Lab1.Core/Contracts/UserResponseTo.cs
using System.Numerics;
namespace Lab1.Core.Contracts
{
    // DTO определяют структуру данных, возвращаемую клиентам
    public record UserResponseTo 
    {
        public long id { get; set; } = 0;
        public string login { get; set; } = string.Empty;
        public string password { get; set; } = string.Empty;
        public string firstname { get; set; } = string.Empty;
        public string lastname { get; set; } = string.Empty;
    }
}

File: 251001/Karpekov/lab1/TaskRest/src/main/java/lab1/modelDTOs/UserResponseTo.java
package lab1.modelDTOs;


public class UserResponseTo {
    private long Id;
    private String Login = "";
    private String Firstname = "";
    private String Lastname = "";

    public UserResponseTo() {
    }

    public UserResponseTo(long id, String login, String firstname, String lastname) {
        Id = id;
        Login = login;
        Firstname = firstname;
        Lastname = lastname;
    }

    public String getFirstname() {
        return Firstname;
    }

    public void setFirstname(String firstname) {
        Firstname = firstname;
    }

    public String getLastname() {
        return Lastname;
    }

    public void setLastname(String lastname) {
        Lastname = lastname;
    }

    public String getLogin() {
        return Login;
    }

    public void setLogin(String login) {
        Login = login;
    }

    public long getId() {
        return Id;
    }

    public void setId(long id) {
        Id = id;
    }
}
